man/man1/fzf.1

```diff
@@ -155,9 +155,22 @@ the full screen.
 .BI "--min-height=" "HEIGHT"
 Minimum height when \fB--height\fR is given in percent (default: 10).
 Ignored when \fB--height\fR is not specified.
+.TP
+.BI "--layout=" "LAYOUT"
+Choose the layout (default: bottom-up)
+
+.br
+.BR bottom-up "     Show everything from bottom up"
+.br
+.BR top-down "      Show items from top down, prompt and header at the bottom"
+.br
+.BR top-down-above "Show everything from top down"
```

Comment:
Well, I wanted `top-down` to be the new name for `--reverse`, and `top-down-above` to be your new layout. I mean "top-down-above (the prompt and header)". But I can definitely see that the term can be confusing, do you have a better idea?

Reply:
Ah, that's fine, I'll do that.